sonar-application/src/main/java/org/sonar/application/App.java

```diff
@@ -171,15 +171,21 @@ public static void main(String[] args) throws InterruptedException {
     CommandLineParser cli = new CommandLineParser();
     Properties rawProperties = cli.parseArguments(args);
     Props props = new PropsBuilder(rawProperties, new JdbcSettings()).build();
+    ClusterProperties clusterProperties = new ClusterProperties(props);
+    clusterProperties.populateProps(props);
+
     AppFileSystem appFileSystem = new AppFileSystem(props);
     appFileSystem.verifyProps();
     AppLogging logging = new AppLogging();
     logging.configure(props);
+    clusterProperties.validate();
```

Comment:
Yep, beacuase AppLogging must be initialized between the loading of properties by the Cluster (in order to have the default values) and the validation because this one must log information because of usage of Sockets to check parameters.